xija/limits.py

```diff
@@ -0,0 +1,60 @@
+def get_limit_spec(limit):
+    """
+    Based on an input *limit* string of the form:
+
+    <system>.<type>.<direction>.<qualifier>
+
+    (where the "qualifier" element is optional) return the various pieces 
+    into a dictionary for convenience.
+    """
+    words = limit.split(".")
+    if len(words) < 3:
+        raise RuntimeError(f"{limit} is not a valid limit string!")
```

Comment:
No exclamation point. I mostly follow the Python builtin exceptions and start with lower case and do not end with any punctuation.

Reply:
Fixed.